IdentityCore/src/oauth2/MSIDRequestParameters.h

```diff
@@ -40,5 +40,7 @@
 
 @property (readwrite) NSString *loginHint;
 @property (readwrite) NSString *extraQueryParameters;
+@property (readwrite) NSString *promptBehavior;
```

Comment:
I think prompt behavior was something AAD V1 specific?

Reply:
It also exists for MSAL in a different value.

        case MSALForceLogin : return @{ @"prompt" : @"login" };
        case MSALForceConsent : return @{ @"prompt" : @"consent" };
        case MSALSelectAccount : return @{ @"prompt" : @"select_account" };